test/liquidation/makeLiquidatableProtocol.ts

```diff
@@ -0,0 +1,201 @@
+import hre, { ethers } from 'hardhat';
+import { exp, setTotalsBasic } from '../helpers';
+import { HttpNetworkConfig } from 'hardhat/types/config';
+import {
+  CometExt__factory,
+  CometHarness__factory,
+  CometHarnessInterface,
+  Liquidator__factory
+} from '../../build/types';
+import {
+  COMP,
+  COMP_USDC_PRICE_FEED,
+  DAI,
+  DAI_USDC_PRICE_FEED,
+  DAI_WHALE,
+  LINK,
+  LINK_USDC_PRICE_FEED,
+  SWAP_ROUTER,
+  UNI,
+  UNI_USDC_PRICE_FEED,
+  USDC,
+  USDC_USD_PRICE_FEED,
+  USDC_WHALE,
+  WBTC,
+  WBTC_USDC_PRICE_FEED,
+  WETH9,
+  ETH_USDC_PRICE_FEED,
+  UNISWAP_V3_FACTORY
+} from "./addresses";
+import daiAbi from './dai-abi';
+import usdcAbi from './usdc-abi';
+
+export default async function makeLiquidatableProtocol() {
```

Comment:
@antonina-cherednichenko I thought we might be able to use this in `test/liquidation/liquidation-bot.ts` as well.

would it put you into rebase hell if I made that change in this branch, too?

Reply:
it shouldn't, let's merge in